descwl_shear_sims/tests/_wcs.py

```diff
@@ -12,7 +12,9 @@ def make_sim_wcs(dim):
 
     dims = [dim]*2
     cen = (np.array(dims)-1)/2
-    image_origin = galsim.PositionD(x=cen[1], y=cen[0])
+    # this is the center in 0 offset
+    # need to add 1 for the galsim convention
+    image_origin = galsim.PositionD(x=cen[1]+1, y=cen[0]+1)
```

Comment:
So this bug in our test sims ensured we missed the offset bug in the main code?

Reply:
I took a look at where this function is referenced. This function is used in `test_dmwcs.py`, `test_dmpsfs.py`, and `test_rand_psf.py`. 

In `test_dmwcs` in `test_dmwcs.py`, both `galsim` and DM WCSs use the wrong center since the DM WCS is produced from the galsim WCS. Therefore they agree with each other. 

I found this since in `make_coadd_dm_wcs` the center is calculated with the DM `bbox.getCenter()` method, which calculates the correct center that can be compared to the wrong `galsim` center. 

In `test_dmpsfs.py` and `test_rand_psf.py`, no DM WCS is used. The image is centered at the wrong image position but doesn't impact the test.